packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_table_v2.dart

```diff
@@ -243,75 +252,104 @@ class LoggingSettingsDialogV2 extends StatelessWidget {
             'General',
           ),
           const StructuredErrorsToggle(),
+          const SizedBox(height: defaultSpacing),
+          _RententionLimitSetting(
+            theme: theme,
```

Comment:
don't pass the theme around. Look this up in the _RententionLimitSetting build method

Reply:
oops forgot to clean this up when the tooling automatically added it.